exporters/tokens/src/helpers/colorHelper.ts

```diff
@@ -73,3 +75,21 @@ export const transformColorsToVariables = (
 
   return transformedValue;
 };
+
+export const filterColorCollections = (tokens: Token[]) => {
+  const COLLECTION_NAME = 'Theme tokens';
```

Comment:
**suggestion (if-minor):** Shouldn't we import this from some kind of config file? As I understand this correctly this is the same name in Supernova/Figma that can change in a future, isn't it?

Reply:
hm, i can move it to the `config.local.json` file. 
There is the `generateOriginalDataFiles` already.

I'm just not sure if it should be configurable or look like it's configurable. For me it's more like a constant/rule how tokens should be organised in Supernova/Figma everytime.

Reply:
Ok, so let's leave it as is.